msgraph/models.go

```diff
@@ -908,7 +909,7 @@ type Group struct {
 	Theme                         *GroupTheme                         `json:"theme,omitempty"`
 	UnseenCount                   *int                                `json:"unseenCount,omitempty"`
 	Visibility                    *GroupVisibility                    `json:"visibility,omitempty"`
-	IsAssignableToRole            *bool                               `json:"isAssignableToRole,omitempty"`
+	WritebackConfiguration        *WritebackConfiguration             `json:"writebackConfiguration,omitempty"`
```

Comment:
Let's name the struct `GroupWriteBackConfiguration` to match the API model

```suggestion
	WritebackConfiguration        *GroupWritebackConfiguration             `json:"writebackConfiguration,omitempty"`
```